scripts/policy/misc/unknown-protocols.zeek

```diff
@@ -0,0 +1,58 @@
+##! This script logs information about packet protocols that Zeek doesn't
+##! know how to process. Mostly these come from packet analysis plugins when
+##! they attempt to forward to the next analyzer, but they also can originate
+##! from non-packet analyzers.
+
+@load base/frameworks/notice
+
+module UnknownProtocol;
+
+export {
+	redef enum Log::ID += { LOG };
+
+	global log_policy: Log::PolicyHook;
+
+	type Info: record {
+		## Timestamp for when the measurement occurred.
+		ts:           time     &log;
+		##
+		analyzer:     string   &log;
+		protocol_id:  string   &log;
+		first_bytes:  string   &log;
+	};
+
+	## How many reports for an analyzer/protocol pair will be allowed to
+	## raise events for logging being rate-limited.
+	option sampling_threshold : count = 25 &redef;
```

Comment:
Think it may be be fine to have a much lower threshold by default.  Maybe like 1-3 really seems enough.

Reply:
Yah the values of these was really up for review here. I picked the defaults used for weirds, but I agree that it can still be a ton of output.